### PR TITLE
new records

### PR DESCRIPTION
**Pull request by thea-m**
Old records, PRS13831WaldaTH and PRS13832WaldaM are doublets (PRS12586WaldaMaryam and https://betamasaheft.eu/persons/PRS12585WaldaTaklaHaymanot/main) and will be deleted after merging

### CODE REVIEW

**Review by nafisa-valieva — APPROVED** [review]
Looks good for me!

File: new/PRS13830MarkVII.xml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+            <change who="DR" when="2022-05-17">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1" sameAs="Q4120617">
+                    <persName xml:lang="en" xml:id="n1">Mark VII</persName>
+                    <occupation type="ecclesiastic">106th Patriarch of Alexandria</occupation>
+                    <floruit notBefore="1745-05-30" notAfter="1769-05-18"/>
```
Comment (CarstenHoffmannMarburg) [inline new/PRS13830MarkVII.xml:50]: If these are the exact dates of his reign, this could be noted explicitely.

File: new/PRS13831WaldaTH.xml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            <change who="DR" when="2022-05-17">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ወልደ፡ ተክለ፡ ሃይማኖት፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Walda Takla Hāymānot</persName>
+                    <floruit notBefore="1769" notAfter="1777"/>
```
Comment (CarstenHoffmannMarburg) [inline new/PRS13831WaldaTH.xml:50]: We used to give the floruits of persons also explicitely.

File: new/PRS13832WaldaM.xml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+            <change who="DR" when="2022-05-17">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ወልደ፡ ማርያም፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Walda Māryām</persName>
+                    <floruit notBefore="1769" notAfter="1777"/>
```
Comment (CarstenHoffmannMarburg) [inline new/PRS13832WaldaM.xml:50]: We used to give the floruits of persons explicitely.

**Pull request merged**